### PR TITLE
Section 1.2. Identity System

### PR DESCRIPTION
**Pull request by jurij-jukic**
clarify domain provenance and resolution

### CODE REVIEW

**Comment by jurij-jukic** [comment]
Accidentally added Section 2.1.2. stuff here with process startup, shutdown and crashes. I will keep it here since we already agreed on everything.

File: src/identity_system.md [hunk 1]
```diff
@@ -33,9 +37,11 @@ These routers are other *direct* nodes that have agreed to forward messages to i
...
 
+## Conclusion
+
 For more information about the architectural specifics of the networking protocol, see [Networking Protocol](./networking_protocol.md).
 The main takeaway for the identity system is that *domain provenance* and *domain resolution* are unified by KNS.
 
 Like .eth for ENS, the KNS domain space is fixed inside the `.os` top-level domain.
-However, we reserve the ability to expand domain availability in the future, and governance of the Kinode protocol will include the ability to manage domain names.
-Eventually, we hope to link various TLDs to existing NFT communities and other identity systems.
+However, we reserve the ability to expand domain availability in the future to domains other than `.os`, and governance of the Kinode protocol will include the ability to manage domain names.
```
Comment (nick1udwig) [inline src/identity_system.md:46]: This sentence has a lot of `-ability`s in close proximity (I know it was that way before you edited -- just seeing it now). Can we change it to read something like
```
However, Kinode DAO has the right to manage domains, including domains other than `.os`, and so domain availability is likely to expand in the future (subject to governance decisions).
```

**Review by nick1udwig — APPROVED** [review]
Recommended tweak on text that was around before. Thanks for the edits: document is much more readable!

File: src/identity_system.md (hunk 1 above)
Comment (jurij-jukic) [reply src/identity_system.md]: done. i just removed 'and' from the sentence because 'and so' seemed informal

**Pull request merged**